src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -132,6 +132,11 @@ class NoTemplatesServiceApi(object):
         and trailing
         whitespace.
 
+        Example:
+          >>> from google.example.v1.no_path_templates_pb2 import NoTemplatesServiceApi
```

Comment:
This is still not right -- the module called "no_path_templates_pb2" is a compiled proto file, and doesn't have the class NoTemplatesServiceApi in it. (I'm still not sure this import make sense, since these examples are within the package you're "importing" from.)

If there is an import here, the package should derived be from the veneer config, and the module should be derived from `getApiWrapperName()`.